docs/standard/data/sqlite/connection-strings.md

```diff
@@ -86,6 +86,15 @@ A value that indicates whether to enable recursive triggers.
 | True  | Sends `PRAGMA recursive_triggers` immediately after opening the connection. |
 | False | Doesn't send `PRAGMA recursive_triggers`. This is the default.              |
 
+### Default Timeout
+
+The default timeout (in seconds) for executing commands. The default value is 30. *Command Timeout* is an alias of this keyword.
+
+This value can be overridden using <xref:Microsoft.Data.Sqlite.SqliteConnection.DefaultTimeout> which in turn can be overridden using <xref:Microsoft.Data.Sqlite.SqliteCommand.CommandTimeout>.
+
+> [!NOTE]
+> The Default Timeout keyword was added in version 6.0.
```

Comment:
```suggestion
> The `DefaultTimeout` property was added in version 6.0.
```

Instead of an alert here at the end, you might want to open this section with this info. For example, "Starting with version 6.0, you can use the `DefaultTimeout` to ..."

Reply:
Again, there are no properties in connection strings--only on .NET types.

Using `NOTE` is consistent with other keywords on this page, and with the EF Core docs. Given that we have three supported versions, I think this information is important to see while scanning.